lib/plugins/rest.js

```diff
@@ -161,6 +161,40 @@ exports.register = function (server, options, next) {
 			});
 		}
 	});
+
+	server.route({
+		method  : "PUT",
+		path    : "/v1/instances/{instanceId}",
+		handler : function (request, reply) {
+			instances.getInstance({ id : request.params.instanceId })
+			.then(function (response) {
+
+				var instance = new Instance({
+					id    : request.params.instanceId,
```

Comment:
Just a thought, but if you renamed `{instanceId}` to `{id}` then you could simply do something like `var instance = new Instance(request.params);`.